integrations/symfony-bundle.rst

```diff
@@ -0,0 +1,176 @@
+Symfony Bundle
+==============
+
+The usage documentation is split into two parts. First we explain how to configure the bundle in an application. The second part is for developing reusable Symfony bundles that depend on an HTTP client defined by the Httplug interface.
+
+For information how to write applications with the services provided by this bundle, have a look at the [Httplug documentation](http://docs.php-http.org).
+
+
+Use in Applications
+-------------------
+
+Custom services
```

Comment:
this title is confusing me. just "Services" would probably be better. or "HTTP Services"